openpype/hosts/maya/plugins/publish/validate_render_image_rule.py

```diff
@@ -37,15 +43,26 @@ def process(self, instance):
     @classmethod
     def repair(cls, instance):
 
-        required_images_rule = cls.get_default_render_image_folder(instance)
+        required_images_rule = cls.get_default_render_image_folder(
+            cls, instance
+        )
```

Comment:
```suggestion
        required_images_rule = cls.get_default_render_image_folder(
        instance
        )
```
it might error out with (cls, instance)

Reply:
Why do you think it'll error?

Reply:
it will error with TypeError for the function missing one required positional argument. `cls.get_default_render_image_folder` expects 1 argument instead of 2.

Reply:
But the function arguments is changed?

Reply:
maybe I am wrong as I notice the` @staticmethod`

Reply:
The code seems to be working here. We could remove the static method decorator?
Not sure why its there anyways.

Reply:
yes, we should remove @staticmethod as we dont need it for this case.

Reply:
It is static method, but we should remove `cls` attribute from `get_default_render_image_folder`.

Reply:
Ok, lets take this back a bit. I need to log a message for the user as debug call. What would be the best route here?

- pass in the log to `get_default_render_image_folder`
- remove the static method decorator and turn `get_default_render_image_folder` into a class method? Why is `get_default_render_image_folder` a static method to begin with?
- alternatives?

Reply:
Following @BigRoy advice. Please unresolve the conversation if you think its not the right direction.